Makefile

```diff
@@ -158,3 +160,16 @@ $(ADDLICENSE): $(LOCALBIN)
 goimports: $(GOIMPORTS) ## Download goimports locally if necessary.
 $(GOIMPORTS): $(LOCALBIN)
 	test -s $(LOCALBIN)/goimports || GOBIN=$(LOCALBIN) go install golang.org/x/tools/cmd/goimports@$(GOIMPORTS_VERSION)
+
+.PHONY: generate-mocks
+generate-mocks: mockgen ## Generate code (mocks etc.).
+	MOCKGEN=$(MOCKGEN) go generate ./...
+
+.PHONY: mockgen
+mockgen: $(MOCKGEN)
+$(MOCKGEN): $(LOCALBIN)
+	test -s $(LOCALBIN)/mockgen || GOBIN=$(LOCALBIN) go install github.com/golang/mock/mockgen@$(MOCKGEN_VERSION)
+
+.PHONY: update-mock
+update-mock: ## Run update-gomock script to update mountutils mock file
+	./hack/update-gomock
```

Comment:
Wouldn't it make sense to move this before we run `go generate ...` in the `generate-mocks` directive?

Reply:
`update-mock` modifies the mock generated by `generate-mocks` so it should be called after `generate-mocks`